spec/views/devise/shared/_password_strength.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe 'devise/shared/_password_strength.html.erb' do
+  let(:forbidden_passwords) { nil }
+
+  before(:each) do
+    allow(view).to receive(:forbidden_passwords).and_return(forbidden_passwords)
```

Comment:
We don't need this --- we just need to pass an argument to `render`, like `locals: { forbidden_passwords: forbidden_passwords }`

Example: https://github.com/18F/identity-idp/blob/master/spec/views/accounts/_nav_auth.html.erb_spec.rb#L14

Reply:
> We don't need this --- we just need to pass an argument to `render`, like `locals: { forbidden_passwords: forbidden_passwords }`

Ah, good to know. I'll update 👍

Reply:
> We don't need this --- we just need to pass an argument to `render`, like `locals: { forbidden_passwords: forbidden_passwords }`
> 
> Example: https://github.com/18F/identity-idp/blob/master/spec/views/accounts/_nav_auth.html.erb_spec.rb#L14

Updated in dc240be / 303c29c3bffce2266e2ea6074c07cefe86bb0d50.